react/client/src/components/RadioGroup.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Radio } from '@material-ui/core';
+
+interface RadioButtonProps {
+  label: string;
+  inputName: string;
+  activeRadio: string;
+  handleChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+const RadioButton = ({
+  label,
+  activeRadio,
+  handleChange,
+  inputName,
+}: RadioButtonProps) => {
+  return (
+    <div>
+      <Radio
+        value={label}
+        color="primary"
```

Comment:
Not sure if it's still necessary, but the figma has the color of the radio button as the bright teal color

Reply:
Nice, Good eye, I was thinking we could finish with the styling in another Issue/PR and have this one just be for logic but I have made that change.